onnxruntime/test/testdata/abs_free_dimensions.onnx

```diff
@@ -0,0 +1,14 @@
+backend-test:s
+
```

Comment:
Is this file added correctly? Usually an onnx file will have binary type and not display as text in the PR.

Reply:
Hmm, interesting. I'm not sure why this is showing  up - it's definitely a binary file. It might just be that the file is small enough that it looks similar to a text file to the diff tool.